ts/src/xt.ts

```diff
@@ -213,6 +214,8 @@ export default class xt extends Exchange {
                         'post': {
                             'order': 0.2,
                             'withdraw': 1,
+                            'balance/transfer': 1,
+                            'balance/account/transfer': 1,
```

Comment:
I will check it, because I think we can probably implement `transfer` with this one

Reply:
I've already checked that it works, yes, good for transfer

Reply:
> I've already checked that it works, yes, good for `transfer`

@ndubel added 🙂